src/affiliation.py

```diff
@@ -45,22 +45,41 @@ def __init__(
         """
         # pylint: disable=too-many-arguments
         self.id = id_
-        self.name = name
-        self.coordinator = coordinator
-        self.coordinator_email = coordinator_email
+        self.name = name if name else ""
+        self.coordinator = coordinator if coordinator else ""
+        self.coordinator_email = coordinator_email if coordinator_email else ""
         self.status = status
-        self.type = type_
-        self.family = family
-        self.members = members
-        self.approvers = approvers
-        self.clinvar_submitter_ids = clinvar_submitter_ids
+        self.type = type_ if type_ else ""
+        self.family = family if family else ""
+        self.members = members if members else []
+        self.approvers = approvers if approvers else []
+        self.clinvar_submitter_ids = clinvar_submitter_ids if clinvar_submitter_ids else []
         self.errors: dict = {}
 
     @classmethod
-    def _row_to_affiliation(cls, row: tuple) -> object:
+    def _row_to_affiliation(cls, row: tuple) -> 'Affiliation':
         """Convert table row to instance of affiliations object."""
         return cls(*row)
 
+    @classmethod
+    def get_by_id(cls, id_) -> Optional['Affiliation']:
+        """Return an affiliation matching a given ID."""
+        con = sqlite3.connect(DB_FILE)  # type: ignore
+        cur = con.cursor()
+        try:
+            cur.execute("SELECT * FROM affiliations WHERE id = ?", (id_, ))
```

Comment:
I believe this is safe based on the [Python sqlite3 tutorial docs](https://docs.python.org/3/library/sqlite3.html#tutorial):

> Notice that `?` placeholders are used to bind data to the query. Always use placeholders instead of [string formatting](https://docs.python.org/3/tutorial/inputoutput.html#tut-formatting) to bind Python values to SQL statements, to avoid [SQL injection attacks](https://en.wikipedia.org/wiki/SQL_injection) (see [How to use placeholders to bind values in SQL queries](https://docs.python.org/3/library/sqlite3.html#sqlite3-placeholders) for more details).